docs/website/docs/guides/ml-frameworks/pytorch.md

```diff
@@ -322,6 +322,132 @@ their values independently at runtime.
         self.value = new_value
     ```
 
+#### :octicons-code-16: Separating parameters during compilation
```

Comment:
nit: This isn't _just_ about compilation, it's also about runtime, since different parameters can be fed to the same loaded program at runtime. Could change the section header to something more like "Using external parameters"